src/modules/reporting/components/reporting-report-form/reporting-report-form.styles.less

```diff
@@ -6,6 +6,24 @@ ul.ReportingReportForm__fields {
   justify-content: flex-start;
 }
 
+.ReportingReport__Form__err {
+  color: @color-negative;
```

Comment:
Can we simplify here by extending FormStyles.Form__error?

Reply:
The BEM model as I understand it is Block__Element--modification. Is error a modification or an element here?